packages/core/src/fhir-deduplication/resources/medication.ts

```diff
@@ -61,6 +61,11 @@ export function groupSameMedications(medications: Medication[]): {
   }
 
   for (const medication of medications) {
+    if (medication.id && medication.code?.text?.toLowerCase().includes("no known")) {
```

Comment:
Can we encapsulate the specific logic about "no known" in a isBlacklisted function?